### ISSUE TITLE
Setup the frontend for Signup screen

### ISSUE THREAD

**Issue by destrex271** [MicrosoftStudentChapter/notebook#3]
Add options for authentication just by Email or via Google or Microsoft Accounts(Single sign-on).

**Comment by devkishor8007** [comment]
@destrex271 could you please provide the UI of this task?

**Comment by destrex271** [comment]
Yes we just added the link to the figma design in the readme file, you can check it out there

**Comment by devkishor8007** [comment]
I have not seen sign-on page where there is Email or via Google or Microsoft Accounts.
If you make complete please let me know!

**Comment by devkishor8007** [comment]
while completed the design, please assign this task to me where I am notify!!
Thank you

**Comment by destrex271** [comment]
Yup the designs will be finished soon. We are trying to provide chances to contribute to as many people as possible, so you can pick up some other issue till then and revert back to this one if you wish to.

**Comment by devkishor8007** [comment]
glad to know!!

**Comment by devkishor8007** [comment]
@destrex271 I have not seen any Signup screen. In the figma design, there is login screen only.
if you made please insert the UI of  Signup screen.
Thank you
